packages/mirrorful/editor/src/components/Layout.tsx

```diff
@@ -0,0 +1,136 @@
+import { Box, Spinner, useDisclosure } from '@chakra-ui/react'
+import { TTab } from '@mirrorful/core/lib/components/Dashboard'
+import { Sidebar } from '@mirrorful/core/lib/components/Dashboard/Sidebar'
+import { ExportSettingsModal } from '@mirrorful/core/lib/components/ExportSettingsModal'
+import { ExportSuccessModal } from '@mirrorful/core/lib/components/ExportSuccessModal'
+import { Onboarding } from '@mirrorful/core/lib/components/Onboarding'
+import { AnimatePresence, motion } from 'framer-motion'
+import { useRouter } from 'next/router'
+import { useEffect, useState } from 'react'
+import postStoreData from 'src/utils/postStoreData'
+import useMirrorfulStore from 'src/zustand/useMirrorfulStore'
+
+type props = { children: React.ReactNode; isLoading?: boolean }
+export function Layout({ children, isLoading = false }: props) {
+  const platform = 'package'
+  const router = useRouter()
+  const [tab, setTab] = useState<TTab>(
+    router.pathname === '/'
+      ? 'colors'
+      : (router.pathname.replace('/', '') as TTab)
```

Comment:
Hm, maybe instead of trying to get the path to be casted as a `TTab` we instead just change "/" to be a valid tab name. I.e., tab names and route names become the same thing.

Not a huge fan of the usage of "as"

Reply:
I might keep this because the web edition still needs TTab. I will add a Todo comment to remove it!